diff --git a/dist/await.d.ts b/dist/await.d.ts
index b9d4e2d6221522d0e9fde4e051560f686f038cec..2c2406391ae43c3ec6f4e0c2357d0ba41c83ff01 100644
--- a/dist/await.d.ts
+++ b/dist/await.d.ts
@@ -13,5 +13,5 @@ export declare const variants: Record<string, string>;
 export declare const license: string;
 export declare const url: string;
 
-declare const await: IconModule;
-export default await;
+declare const awaitIcon: IconModule;
+export default awaitIcon;
diff --git a/dist/await.js b/dist/await.js
index 1097ad07136c07fdd45552448fee22578c6bdf59..7c2dab6fe6fe23d5973e5291ba6ee4b7dae14533 100644
--- a/dist/await.js
+++ b/dist/await.js
@@ -13,5 +13,5 @@ export const variants = {
 export const license = "CC0-1.0";
 export const url = "https://awaitwidget.com";
 
-const await = { slug, title, hex, categories, aliases, svg, variants, license, url };
-export default await;
+const awaitIcon = { slug, title, hex, categories, aliases, svg, variants, license, url };
+export default awaitIcon;
